app/src/main/java/org/kiwix/kiwixmobile/main/MainActivity.java

```diff
@@ -633,6 +651,84 @@ private void updateTitle() {
     }
   }
 
+
+  public static void sendNotification(String message,Context context){
+    Log.d("TAG", "sendNotification: ");
+
+
+    Intent actionIntent_stop = new Intent(context, NotificationActionService.class)
+        .setAction(ACTION_STOP);
+
+    PendingIntent actionPendingIntent_stop = PendingIntent.getService(context, 0,
+        actionIntent_stop, PendingIntent.FLAG_ONE_SHOT);
+
+
+    Intent actionIntent_pause_play = new Intent(context, NotificationActionService.class)
+        .setAction(ACTION_PAUSE);
+
+    PendingIntent actionPendingIntent_pause_play = PendingIntent.getService(context, 0,
+        actionIntent_pause_play, PendingIntent.FLAG_ONE_SHOT);
+
+    NotificationCompat.Builder mBuilder = new NotificationCompat.Builder(context)
+        .setSmallIcon(R.drawable.icon_question)
+        .setContentTitle("Read Aloud")
+        .setVisibility(NotificationCompat.VISIBILITY_PUBLIC)
+        .addAction(R.drawable.ic_stop_icon, "stop",
+            actionPendingIntent_stop)
+        .setPriority(Notification.PRIORITY_MAX);
+
+
+
+if(message.equals("resume")) {
+   mBuilder.addAction(R.drawable.ic_play_icon, message,
+              actionPendingIntent_pause_play);
+
+
+}
+else{
+   mBuilder.addAction(R.drawable.ic_pause_icon, message,
+          actionPendingIntent_pause_play);
+
+}
+
+    NotificationManager mNotificationManager =
+
+        (NotificationManager) context.getSystemService(Context.NOTIFICATION_SERVICE);
+
+
+    mNotificationManager.notify(001, mBuilder.build());
```

Comment:
Do not use Hungarian notations. Follow the contributors' guidelines given in this repo and also reformat you code using Square's code style.